dsc_lib/src/schemas/mod.rs

```diff
@@ -19,15 +19,15 @@ pub enum SchemaUriPrefix {
 impl std::fmt::Display for SchemaUriPrefix {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
-            Self::AkaDotMs => write!(f, "{}", "https://aka.ms/dsc/schemas"),
-            Self::Github => write!(f, "{}", "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas"),
+            Self::AkaDotMs => write!(f, "https://aka.ms/dsc/schemas"),
+            Self::Github => write!(f, "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas"),
         }
     }
 }
 
 impl SchemaUriPrefix {
     /// Returns every known URI prefix for convenient iteration.
-    pub fn all() -> Vec<SchemaUriPrefix> {
+    #[must_use] pub fn all() -> Vec<SchemaUriPrefix> {
```

Comment:
Preference is to have attribute on own line to be consistent with reset of codebase

Reply:
I followed the clippy recommendation from CI test output, will switch.